apps/web/src/components/shared/ProfileImage.tsx

```diff
@@ -9,7 +9,7 @@ interface ProfileImageProps {
   /**
    * **NOTE**: Value should be URL string
    */
-  value: string;
+  value?: string | null;
```

Comment:
Extended it because we support in code both. But mainly needed because the `profilePicture` prop from the dto is defined like that. Otherwise, I would have to do `value={currentUser?.profilePicture ?? ''}` which is a hacky and also will fall under the `false` case of the `ProfileImage` ternary statement.